Konferenco Moderating Monero original transcription (#183)
monero-ecosystem/outreach-docs: monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Moderating_Monero.md

> 	 I Recently created this a discussion thread for the community to actually talk about whether they wanted to allow light altcoin discussion,  so say maybe top 20 by market cap. We still haven't determined. They are a reasonable filter at this point but it's currently under discussion to allow it in other other forums, but that's that's another thing that's on my plate, and I believe that's it. There's just gonna be a copyright.. oh closing thoughts. Perfect. Moderating a community that leans as heavily libertarian and our Anarcho-capitalist as the Monero community does is extremely difficult and has been a huge learning experience. I Found that a light touch is usually best whenever possible, good moderators tend to use their words first and their mod tools' is a last resort, and you have to know the taking action builds resentment, it's easy to go and hit the remove or the ban button dealing with the followed is very difficult a lot of people are willing to wield the power without actually taking responsibility for it afterwards and that's a very difficult thing. 

Reviewer: ```suggestion
	 I recently created this a discussion thread for the community to actually talk about whether they wanted to allow light altcoin discussion, so say maybe top 20 by market cap. We still haven't determined a reasonable filter at this point but it's currently under discussion to allow it in other other forums. But that's another thing that's on my plate, and I believe that's it. There's just gonna be a copyright.. oh closing thoughts. Perfect. Moderating a community that leans as heavily libertarian and our Anarcho-capitalist as the Monero community does is extremely difficult and has been a huge learning experience. I found that a light touch is usually best whenever possible, good moderators tend to use their words first and their mod's tool is a last resort, and you have to know the taking action builds resentment. It's easy to go and hit the remove or the ban button dealing with the followed is very difficult. A lot of people are willing to wield the power without actually taking responsibility for it afterwards and that's a very difficult thing. 
```

	Anyways, that's about it for my talk. If anybody has any questions on Monero�s moderation I'm here for the rest of the evening.
 

> Surae: Let�s thank needmoney again, and let's actually� but we have questions and ones right here

Reviewer: ```suggestion
Surae: Let’s thank needmoney again, and let's actually... we just applauded him but we don't have to thank him yet, but we have questions and ones right here.
```

Audience member:  Thank you really cool talk, that's on right? And thanks for your service as a moderator. You've been a good one, it's good to know that we can we have someone in there we can trust. Do you, like in your experiences do you think that you've been able to distill any notion of a bad actor? And what would constitute a bad actor or is there any such a thing? 

> Needmoney90: Only by gut feeling. Unfortunately, if I was to try and create any rigorous definition for it, there's always something that would slip through, we have to take it on a case-by-case basis 

Reviewer: ```suggestion
Needmoney90: Only by gut feeling. Unfortunately, if I was to try and create any rigorous definition for it, there's always something that would slip through, we have to take it on a case-by-case basis.
```

> Needmoney90: Okay, thank you... 

Reviewer: ```suggestion
Surae: We have one back there... Does anybody else have any other questions for needmoney?

Needmoney90: Okay, thank you... 
```

Surae: Actually don't walk away yet. I have a quick question for you. Yeah, so you mentioned that already that basically it comes down to gut instinct and trying to manage the blowback versus�

> Needmoney90: After you have a gut feeling about something at the talk to the person and just actually have a dialogue and very quickly you can figure out whether they're willing to listen or whether they're just going to talk past you and I mean if they're talking past you and it's very obvious you can tell what's going on. 

Reviewer: ```suggestion
Needmoney90: You talk to the person, after you have a gut feeling...

Surae: Ok. Go ahead, sorry.
 
Needmonay90: After you have a gut feeling about something at the talk to the person and just actually have a dialogue and very quickly you can figure out whether they're willing to listen or whether they're just going to talk past you. And I mean if they're talking past you and it's very obvious you can tell what's going on. So, yeah.
```

> Surae: Honestly, I'm extremely grateful personally on a couple of levels for having needmoney in the community moderating the way that he does he does a fantastic job, and I want to thank him again.

Reviewer: ```suggestion
Surae: Alright everybody. Thanks again for needmoney. Honestly, I'm extremely grateful personally on a couple of levels for having needmoney in the community moderating the way that he does. He does a fantastic job, and I want to thank him again.
```
